### PR TITLE
Initramfs fsck

### PR DESCRIPTION
**Pull request by JoergZeidler**

### CODE REVIEW

**Review by HarryWaschkeit — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [kirkstone-0.27.2] Q4 2023
+- initramfs:
+  - fixed fsck vfat handling
```
Comment (mlilien) [inline CHANGELOG.md:10]: how?

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount [hunk 2]
```diff
@@ -30,7 +30,7 @@ run_fs_mount()
 
     # mount boot
     check_fs "boot"
-    mount /dev/omnect/boot "${ROOTFS_DIR}/boot" || return 1
+    mount -t vfat /dev/omnect/boot "${ROOTFS_DIR}/boot" || return 1
```
Comment (mlilien) [inline recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount:33]: why do we need that? what if we have a target with a non vfat boot partition?

File: CHANGELOG.md (hunk 1 above)
Comment (JoergZeidler) [reply CHANGELOG.md:10]: vfat was not working for u-boot; fixed via added dosfstools.....

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount (hunk 2 above)
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount:33]: in grub context we have mount the boot partition with the type option vfat as well. could remove it here anyway.

Comment (mlilien) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount:33]: please do.

File: CHANGELOG.md (hunk 1 above)
Comment (mlilien) [reply CHANGELOG.md:10]: would be nice what exactly was fixed in the changelog

Comment (JoergZeidler) [reply CHANGELOG.md]: done

File: recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount (hunk 2 above)
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/fs-mount]: done

**Review by mlilien — APPROVED** [review]

**Pull request merged**